test/e2e/framework/auth/helpers.go

```diff
@@ -122,10 +143,11 @@ func bindInNamespace(c v1beta1rbac.RoleBindingsGetter, roleType, role, ns string
 		Subjects: subjects,
 	})
 
-	// if we failed, don't fail the entire test because it may still work. RBAC may simply be disabled.
 	if err != nil {
-		klog.Errorf("Error binding %s/%s into %q for %v\n", roleType, role, ns, subjects)
+		return fmt.Errorf("error binding %s/%s into %q for %v: %v", roleType, role, ns, subjects, err)
```

Comment:
Added the err to the message here; not sure why it wasn't included in the first place.

Reply:
Why not use errors.Wrapf then here.

Reply:
You're right, I should have. Will fix.